unstructured_api_tools/pipelines/templates/pipeline_api.txt

```diff
@@ -126,9 +132,9 @@ def ungz_file(file: UploadFile) -> UploadFile:
         file=io.BytesIO(gzip_file.read()),
         size=len(gzip_file.read()),
         filename=filename[:-3] if len(filename) > 3 else "",
```

Comment:
i think this is to cover the case `filename.endswith(".gz")`, so the condition may as well be updated to reflect that.